keadm/README.md

```diff
@@ -1,17 +1,258 @@
-# KubeEdge Installer
+# Getting Started with KubeEdge Installer
 
-[Kubeedge Installer Doc](<https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/kubeedgeadm-scope.md/>)
+Please refer to KubeEdge Installer proposal document for details on the motivation of having KubeEdge Installer.
+It also explains the functionality of the proposed commands.
+[KubeEdge Installer Doc](<https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/kubeedgeadm-scope.md/>)
 
-#### Build KubeEdgeinstaller
+## Limitation
 
- ```shell
-    cd $GOPATH/src/github.com/kubeedge/kubeedge/keadm
-    
-    make
-     
-    sudo mv keadm /usr/local/bin
- ```
-  
-#### Execute command to get started
+- Currently support of `KubeEdge installer` is available only for Ubuntu OS. CentOS support is in-progress.
 
-sudo keadm --help
\ No newline at end of file
+## Downloading KubeEdge Installer
+
+1. Goto [KubeEdge Release](<https://github.com/kubeedge/kubeedge/releases>) page and download `keadm-$VERSION-$OS-$ARCH.tar.gz.`.
+2. Untar it your desired folder, by executing `tar -xvzf keadm-$VERSION-$OS-$ARCH.tar.gz`.
+3. The binary is untarred in `kubeedge` folder in current path. The binary name is `kubeedge`
+
+## Building from source
+
+1. Download the source code either by
+
+- `git clone https://github.com/kubeedge/kubeedge.git $GOPATH/src/github.com/kubeedge/kubeedge`
+2. `cd $GOPATH/src/github.com/kubeedge/kubeedge/keadm`
+3. `make`
+4. Binary `kubeedge` is available in current path
+
+## Installing KubeEdge Cloud component
+
+Referring to `KubeEdge Installer Doc`, the command to install KubeEdge cloud component (edge controller) and pre-requisites
+
+- Execute `sudo kubeedge init`
```

Comment:
please remove sudo in all commands and mention at one place that root permissions are required to run the binary

Reply:
installer runs fine with both root and sudo user..But now i will remove sudo ..And in the Note section i will put , if user wants to run as sudo user he can add sudo command in front of the command which he wants to execute.